test/failing/tests/unit_modtypes.ml.broken-ref

```diff
@@ -2,8 +2,4 @@ ocamlformat: ignoring "tests/unit_modtypes.ml" (syntax error)
 File "tests/unit_modtypes.ml", line 66, characters 13-14:
 66 |   module M (X) (Y):
                   ^
-Error: Syntax error: 'end' expected
-File "tests/unit_modtypes.ml", line 34, characters 0-3:
-34 | sig
-     ^^^
-  This 'sig' might be unmatched
+Error: Syntax error
```

Comment:
Why are errors printed differently ?

Reply:
This is due to https://github.com/ocaml/ocaml/pull/10095